src/index.html

```diff
@@ -6,9 +6,85 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner">
+      <a
+        class="logo"
+        href="http://localhost:61118/">
+        <img
+            src="./images/logo.png"
+            alt="logo"
+            href="#logo">
```

Comment:
```suggestion
        <img
           src="./images/logo.png"
           alt="logo"
           href="#logo"
         >
```

Reply:
close opening tag on new line under tag start
attributes should have 2-space indentation

Fix everywhere please